pages/dkp/kommander/2.2/appdeployments/index.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: layout.pug
+navigationTitle: AppDeployments
+title: AppDeployments
+excerpt: Use AppDeployments to deploy, and customize platform, DKP catalog, and custom applications in your environment
+menuWeight: 50
+beta: false
+enterprise: false
+---
+
+An `AppDeployment` is a [Custom Resource][CRD] created by DKP with the purpose of deploying applications (platform, DKP catalog and custom applications) in the management cluster, managed clusters, or both. Customers of both Essential and Enterprise products use `AppDeployments`, regardless of their setup (networked, air-gapped, etc.), and their infrastructure provider.
+
+When installing DKP, an `AppDeployment` resource is created for each enabled **platform application**. This `AppDeployment` resource references a `ClusterApp`, which then references the repository that contains a concrete declarative and preconfigured setup of an application, usually in the form of a [`HelmRelease`][helm].
+
+In the case of **DKP catalog** and **custom applications**, the `AppDeployment` references an `App` instead of a `ClusterApp`, which also references the repository containing the installation and deployment information.
```

Comment:
```suggestion
In the case of **DKP catalog** and **custom applications**, the `AppDeployment` references an `App` instead of a `ClusterApp`, which also references the repository containing the installation and deployment information. `Apps` are namespace-scoped and are meant to only be deployable to the workspace or project in which they have been created.
```